### ISSUE TITLE
Feature Request: Change icon size per icon type

### ISSUE THREAD

**Issue by trippyhat** [AHAAAAAAA/PokemonGo-Map#2328]
It's a bit straining to view pokemons when pokestops and gyms are all the same icon size. 

It would be great if we could declare in the config file (or through console args) a way to set the size for each icon type. 

ie,
```
icons_pokestop: small
icons_gym: small
icons_pokemon: normal
```
or
```
-is small -ig small -ip normal
```

**Comment by jnovack** [comment]
This is a front-end preference, not a config change that would require one to restart the service.

**Comment by trippyhat** [comment]
@jnovack Yes, but it should also be configurable through the args and config file. Just like how you can disable display of the pokestops/gyms as a map option, but it's also loaded on start through args/config file.

**Comment by jnovack** [comment]
I wholly and completely disagree.  Pokestops and Gym processing is server side, disabling them has a server-side benefit.

This proposal as it stands:

* has NO server-side benefit
* requires a RESTART of the application to change
* LOCKS the view to the icon sizes, meaning that your desktop and mobile have to view the same way.
* ADDS even more configuration options to an already over-populated configuration file.

**Comment by andrewda** [comment]
I agree with @jnovack. Someone can add an option in the options menu instead.
